### PR TITLE
Operation/initial antenna tasks

### LINKED ISSUE #144: Antenna
This module will be everything antenna

This module should do the following 

- [x] Check deployment status of the antenna 
- [x] Deploy antenna
- [x] Collect telemetry data
- [x] Collect temperature data
- [x] Pet antenna watchdog

### PR DESCRIPTION
**Pull request by ATGr8**
PR for temperature function

### CODE REVIEW

File: radsat-sk/operation/subsystems/RAntenna.c [hunk 1]
```diff
@@ -26,6 +27,8 @@ static int antennaInitialized = 0;
 /** Track Number of times the antenna attempted to deploy*/
 static int antennaDeploymentAttempts = 0;
 
+/** Index of the Antenna*/
```
Comment (TheDopplerEffects) [inline radsat-sk/operation/subsystems/RAntenna.c:31]: /* Index of the Antenna */

File: radsat-sk/operation/subsystems/RAntenna.c [hunk 2]
```diff
@@ -282,3 +285,31 @@ int antennaTelemetry(antenna_telemetry_t* telemetry) {
 
 	return 0;
 }
+
+/**
+ * Gives temperature of both sides of the antenna
+ * @return 0 for success, non-zero for failure. See hal/errors.h for details.
```
Comment (TheDopplerEffects) [inline radsat-sk/operation/subsystems/RAntenna.c]: Add empty line

**Comment by TheDopplerEffects** [comment]
I resolved some Merging issues and check to make sure they are correct and resolve the issues i found

File: radsat-sk/operation/subsystems/RAntenna.c (hunk 1 above)
Comment (ATGr8) [reply radsat-sk/operation/subsystems/RAntenna.c:31]: All other defines in the files were done with two "*"

**Comment by ATGr8** [comment]
I think this has caused some issue because I am not able to push my commits, we can tackle this tomorrow in person

File: radsat-sk/operation/subsystems/RAntenna.c (hunk 1 above)
Comment (TheDopplerEffects) [reply radsat-sk/operation/subsystems/RAntenna.c:31]: your right

File: radsat-sk/operation/subsystems/RAntenna.c [hunk 3]
```diff
@@ -285,21 +287,25 @@ int antennaTelemetry(antenna_telemetry_t* telemetry) {
 }
 
 /**
+
  * Resets microcontroller of the antenna
  * Cancels the current deployement of the Antenna systems
  * @return 0 for success, non-zero for failure. See hal/errors.h for details.
  */
```
Comment (TheDopplerEffects) [inline radsat-sk/operation/subsystems/RAntenna.c]: What I meant by add white space is that your func doc isn't formatted like the ones found on the [readme](https://github.com/USST-RADSAT-SK/software-and-command#functions). we have two types of func docs [pick one here](https://github.com/USST-RADSAT-SK/software-and-command#functions).

**Pull request merged**